src/app/pages/login-page/login-page.component.ts

```diff
@@ -22,7 +23,7 @@ export class LoginPageComponent {
         return;
       }
 
-      Firebase.getInstance().initialize(firebaseLink);
+      Api.getInstance().initialize(firebaseLink);
```

Comment:
### ❗ Issue: Change Api initialization

This is strange. Maybe delete it? We already have api initialized into `app.component`